homeassistant/components/lektrico/sensor.py

```diff
@@ -113,6 +113,10 @@ class LektricoSensorEntityDescription(SensorEntityDescription):
             "em_offline",
             "em",
             "ocpp",
+            "overtemperature",
+            "switching_phases",
+            "1p_charging_disabled",
+            "unknown",
```

Comment:
We should not add `unknown`, we should return `None` if that's the case. That will render the entity as `unknown`, otherwise there are 2 `unknown` states to pick from and that's bad

Reply:
it should return `None` instead of the string `unknown`

Reply:
I removed `unknown`  and used  `None`, thank you.